k8tls-job/templates/k8tls-job.yaml

```
@@ -12,7 +12,7 @@ spec:
       containers:
       - image: accuknox/accuknox-job:latest
         command: ["/bin/sh", "-c"]
-        args: ['curl --location --request POST "https://${URL}/api/v1/artifact/?tenant_id=${TENANT_ID}&data_type=K8TLS&save_to_s3=false" --header "Tenant-Id: ${TENANT_ID}" --header "Authorization: Bearer ${AUTH_TOKEN}" --form "file=@\"/data/report.json\"" && cat /data/report.json']
+        args: ['./curl_command.sh']
```

data_type is hardcoded to KB in the curl_command.sh script.

---

```suggestion
curl --location --request POST "https://${URL}/api/v1/artifact/?tenant_id=${TENANT_ID}&data_type=${DATA_TYPE}&label_id=${LABEL_NAME}&save_to_s3=true" \
```
you can pass the type as variable